themes/aframe/layout/partials/blog/awoa_contributions.ejs

```diff
@@ -6,6 +6,11 @@
         <%- blog_attribution(contribution.author) %>
         <%- github_contribution(contribution.github, contribution.action) %>
         <%- contribution.description %>
+
+        <% var project = contribution.github.split('|'); %>
+        <% if (project.length > 1) { %>
+          (for <a href="https://github.com/<%- project[0] %>"><%- project[0] %></a>)
```

Comment:
I'll add a comment. It's right, it's for contributions that aren't the A-Frame repo (like contributions to prominent side projects).